### ISSUE TITLE
Multiple Ribbons in same application

### ISSUE THREAD

**Issue by markwilliams1234** [TurboPack/RibbonFramework#83]
I have a need for two or three different ribbons in one application. The Ribbon Designer is not designed to work this way as far as I can see. It works on a single hardcoded ribbon resource "APPLICATION" which means the same ribbon gets added to all forms in an app. I have made some changes to the ribbon designer which allows for the setting of different resource names. 

There is also the issue of unique IDs within each ribbon resource. This is problematic as the designer is currently written. You would need to set every single ID manually. I have written a small function which requests a starting ID and then updates every single resource id for that ribbon so it is possible to avoid conflicts. 

I have also added a function to remove all images with one click rather than one by one.

Would these code changes be useful?

**Comment by joachimmarder** [comment]
> Would these code changes be useful?

Yes, most likely. Could you send a pull request so that I can review and test the changes?

**Comment by markwilliams1234** [comment]
I have added 4 pull requests! Couldn't work out how to change 4 files with one pull request. I also merged my last pull request, thinking it would merge with my other pull requests. I didn't realise it committed to the original. I have restored the original, but my pull request remains cllosed. I have tried to reopen it, but that function doesn't seem to work. The closed pull request relates to some minor changes to ribboncompiler. Basically passing in a resourcename parameter for the compile command. Apologies for the hotch potch of requests.
